tests/views/test.pay-methods.jsx

```diff
@@ -23,11 +23,18 @@ describe('PayMethods', function() {
       },
     ];
 
+    this.getPayMethodsStub = sinon.stub();
```

Comment:
to be consistent it would make sense to call this `fakeGetPayMethods` or `fakePayMethodGetter`